session.go

```diff
@@ -76,8 +76,12 @@ type session struct {
 	runClosed chan struct{}
 	closed    uint32 // atomic bool
 
-	undecryptablePackets []*receivedPacket
-	aeadChanged          chan protocol.EncryptionLevel
+	// when we receive too many undecryptable packets during the handshake, we send a Public reset
+	// but only after a time of protocol.PublicResetTimeout has passed
+	undecryptablePackets               []*receivedPacket
+	receivedTooManyUndecrytablePackets time.Time
```

Comment:
I don't like the variable name, it sounds like a bool. Would you mind adding "Time" as a suffix?